ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -480,6 +480,7 @@ def test_torch_mean(
     backend_fw,
 ):
     input_dtype, x, axis, *_ = dtype_and_x
+    assume("float" in input_dtype[0] or "complex" in input_dtype[0])
```

Comment:
why does it need to assume this when the dtypes are already specified to be `float_and_complex`?
ideally we shouldn't need filtering etc. any dtypes in test but rather on the functions with un/supporteded_dtype decorators.

Reply:
Sorry, my bad. I've made the corrections.